spec.html

```diff
@@ -15528,8 +15531,7 @@ <h1>Static Semantics: PropertyNameList</h1>
         <emu-alg>
           1. Let _list_ be PropertyNameList of |PropertyDefinitionList|.
           1. If PropName of |PropertyDefinition| is ~empty~, return _list_.
-          1. Append PropName of |PropertyDefinition| to the end of _list_.
-          1. Return _list_.
+          1. Return the list-concatenation of _list_ and &laquo; PropName of |PropertyDefinition| &raquo;.
```

Comment:
I would prefer to save the result of `PropName of |PropertyDefinition|` and use it twice.

Reply:
That's orthogonal to this PR, but I'm okay with including it.

Reply:
(I found 3 similar cases.)

Reply:
What I'm saying is I'd prefer not to apply an SDO inside the double angle brackets. Since we're already applying it in the prior step, we can alias it and reference the alias twice. But even if we weren't, I would want to alias it and reference the alias once.

Separately, I would support a PR that uses an alias to de-dupe the other 3 duplicate SDO applications you've found.

Reply:
> What I'm saying is I'd prefer not to apply an SDO inside the double angle brackets.

Ah, got it.

> Separately, I would support a PR that uses an alias to de-dupe the other 3 duplicate SDO applications you've found.

Well, one of the others involves "PropName of |ClassElement|" in angle-brackets, so you'd want that changed here anyhow.

I've handled all 4 as a "clean-up" commit, but if you want, I could instead rebase it into a "preliminary" commit, so that the SDO-invocations-in-angle-brackets never occur (even transiently).